src/styles/blocks/_stars.scss

```diff
@@ -0,0 +1,31 @@
+@import "../utils/variables";
+
+.stars {
+  display: flex;
+  width: 166px;
+  flex-direction: row;
+  align-items: center;
+  position: relative;
+
+  &__star {
+    width: $Size-casual;
+    height: $Size-casual;
+    flex-shrink: 0;
+    position: relative;
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    margin-right: 4px;
+  }
+
+  // Sorry, i dont get it.
+  // You mant the selector "nth-child" to be inside of modifier, not next to?
+  // (did that)
+  // Or you want it to strictly be named N, not 4?
```

Comment:
That's not the best way to have a coversation 😆 Next time simply answer my comment on github.

I want you to implement it the same way you did in layout stars & product card.
The number of active stars should depend on the modifier value (`stars--N`). So when the value is `stars--5` - you want to display 5 active stars. When the value is `stars--2` then 2 and so on.

Reply:
Ok, now i get it xD Sorry for confusion